### DISCUSSION TITLE
Please Help Me With Adding Personal Portfolio To Github Pages

### DISCUSSION THREAD

**Discussion by shakomoss** [community/community#140714] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Hello everyone, I just uploaded my personal portfolio files to Github but when I run it with Github pages, only HTML seems to be there without any CSS and JS

this is the repository - https://github.com/shakomoss/Personal-Portfolio-Website

and this is the github pages deployed - https://shakomoss.github.io/Personal-Portfolio-Website/

can someone who has experience tell me what is going on.

**Comment by Krishneshwaran** [comment]
# Fixing File Linking Issues

It looks like the issue lies in how you're linking your CSS, JavaScript, and image files. Adjusting these links should help resolve the problem.

## In your `index.html` file:

### Linking CSS
```markdown
<link rel="stylesheet" href="style/style.css" type="text/css">
```

### Linking JavaScript
```html
<script src="script/script.js"></script>
```

### Linking Images
```html
<img src="./style/img/nastya-dulhiier-DFrlRyLhCOQ-unsplash.jpg" alt="">
<img src="./style/img/flame.png" alt="flame icon">
<img src="./style/img/shalva-mosiashvili.jpeg" alt="Shalva Mosiashvili Pic" id="shago">
```

Make sure that the images, CSS, and JavaScript files are located in the correct directories as specified in your code.
```
```
/your-project-directory
│
├── /style
│   ├── style.css
│   └── /img
│               ├── nastya-dulhiier-DFrlRyLhCOQ-unsplash.jpg
│               ├── flame.png
│               └── shalva-mosiashvili.jpeg
│
├── /script
│   └── script.js
│
└── index.html
```

Feel free ask!
